database/postgres/dal/game.py

```diff
@@ -34,31 +34,37 @@ def create_game(self, games_dto: GameDTO) -> Games:
             date=games_dto.date,
             api_id=games_dto.api_id
         )
-        self.db_session.add(new_game)
-        self.db_session.commit()
-        self.db_session.refresh(new_game)
+        self.session.add(new_game)
+        self.session.commit()
+        self.session.refresh(new_game)
```

Comment:
empty line after this

Reply:
please add an empty line here. Its absence makes me wanna murder Taras for no reason. I would like to avoid unnecessary casualties if possible